pom.xml

```diff
@@ -651,12 +651,6 @@
         <artifactId>commons-beanutils</artifactId>
         <version>1.9.4</version>
       </dependency>
-      <dependency>
-        <groupId>commons-logging</groupId>
-        <artifactId>commons-logging</artifactId>
-        <!-- Hive uses commons-logging 1.1.3 from 0.13 to 1.2 -->
```

Comment:
For the record, since Apache Hive 2.0.0 removed direct commons-logging dependency via [HIVE-12237 Use slf4j as logging facade](https://issues.apache.org/jira/browse/HIVE-12237), we don't need this.